pysb/tests/test_importers.py

```diff
@@ -67,7 +67,7 @@ def bngl_import_compare_simulations(bng_file, force=False,
             renamed_species = 'Obs_{}'.format(species)
         logger.debug(yfull2[renamed_species])
         assert numpy.allclose(yfull1[species], yfull2[renamed_species],
-                              atol=precision, rtol=precision)
+                              atol=precision*10, rtol=precision*10)
```

Comment:
Can you explain this a bit more? Maybe changing the default `1e-12` to `1e-11` here (and on line 189) is a cleaner option. Also the values in the `REDUCED_PRECISION` dict above might need to be changed too.

Reply:
looks like only the repressilator example was failing so I added a specific case to `REDUCED_PRECISION` and reverted the above change.